app/src/webkit/java/org/mozilla/focus/webkit/FocusWebViewClient.java

```diff
@@ -35,11 +38,16 @@
  * and external URL handling.
  */
 /* package */ class FocusWebViewClient extends TrackingProtectionWebViewClient {
-    private final static String ERROR_PROTOCOL = "error:";
+    private static final String ERROR_PROTOCOL = "error:";
```

Comment:
\[PMD\] INFO: The class 'FocusWebViewClient' has a Cyclomatic Complexity of 3 \(Highest = 6\)\.